xsamtools/cram.py

```diff
@@ -46,12 +46,22 @@ def read_fixed_length_cram_file_definition(fh: io.BytesIO):
     """
     return {
         'cram': fh.read(4).decode('utf-8'),
-        'major_version': int.from_bytes(fh.read(1), byteorder='little'),  # order shouldn't matter with 1 byte
-        'minor_version': int.from_bytes(fh.read(1), byteorder='little'),  # but it's required
+        'major_version': next_int(fh),
+        'minor_version': next_int(fh),
         'file_id': fh.read(20).decode('utf-8')
     }
 
+def int32(fh: io.BytesIO) -> int:
```

Comment:
Naming is inconsistent. I’d suggest `get_int32` and `get_int8`

Reply:
Good point!  I renamed them `decode_int32` and `decode_int8` to keep them consistent with the LTF and ITF functions.